src/components/screens/wallet/delegateProfile/index.js

```diff
@@ -1,47 +1,113 @@
-// istanbul ignore file
+import React, { useEffect } from 'react';
 import { withTranslation } from 'react-i18next';
-import { getNextForgers } from '../../../../utils/api/delegates';
-import { getBlocks } from '../../../../utils/api/blocks';
-import { getTransactions } from '../../../../utils/api/transactions';
-import DelegateProfile from './delegateProfile';
-import transactionTypes from '../../../../constants/transactionTypes';
-import withData from '../../../../utils/withData';
-import { getAPIClient } from '../../../../utils/api/lsk/network';
+import { useSelector } from 'react-redux';
+import Box from '../../../toolbox/box';
+import BoxHeader from '../../../toolbox/box/header';
+import BoxContent from '../../../toolbox/box/content';
+import { DateTimeFromTimestamp } from '../../../toolbox/timestamp';
+import VoteWeight from '../../../shared/voteWeight';
+import { formatAmountBasedOnLocale } from '../../../../utils/formattedNumber';
+import { tokenMap } from '../../../../constants/tokens';
+import LiskAmount from '../../../shared/liskAmount';
+import styles from './delegateProfile.css';
 
-const apis = {
-  delegate: {
-    apiUtil: (liskAPIClient, params) => getAPIClient(liskAPIClient).delegates.get(params),
-    defaultData: {},
-    getApiParams: (state, ownProps) => ({
-      address: ownProps.address,
-    }),
-    transformResponse: response => response.data[0],
-  },
-  lastBlock: {
-    apiUtil: getBlocks,
-    defaultData: false,
-    transformResponse: response => (response.data[0] && response.data[0].timestamp),
-  },
-  txDelegateRegister: {
-    apiUtil: (apiClient, params) => getTransactions(params),
-    getApiParams: (state, ownProps) => ({
-      token: state.settings.token.active,
-      address: ownProps.address,
-      networkConfig: state.network,
-      type: transactionTypes().registerDelegate.outgoingCode,
-      limit: 1,
-    }),
-    defaultData: false,
-    transformResponse: response => (response.data[0] && response.data[0].timestamp),
-  },
-  nextForgers: {
-    apiUtil: getNextForgers,
-    defaultData: [],
-    autoload: true,
-    getApiParams: () => ({
-      limit: 101,
-    }),
-  },
+const delegateProfile = ({
```

Comment:
Can you please move this component into its own file and export the component with the `withTransaction` HOC from the index file?